ouroboros-consensus/src/Ouroboros/Consensus/Storage/VolatileDB/FileInfo.hs

```diff
@@ -38,7 +39,7 @@ import           Ouroboros.Consensus.Storage.VolatileDB.Types
 -- | The internal information the db keeps for each file
 data FileInfo blockId = MkFileInfo {
       fLatestSlot :: !MaxSlotNo
-    , fContents   :: !(Map BlockOffset (FileBlockInfo blockId))
+    , fContents   :: !(Map Word64 (FileBlockInfo blockId))
```

Comment:
I keep this `Word64` because this will be a big Map (1000-10000 entries) and all the wrapping and unwrapping could be costly. And also `FileInfo` is opaque.